Sources/DataPipeline/CIODataPipelineEvents.swift

```diff
@@ -0,0 +1,116 @@
+import CioInternalCommon
+import Foundation
+
+// MARK: - Typed Event Signatures
+
+public extension CustomerIO {
+    // make a note in the docs on this that we removed the old "options" property
+    // and they need to write a middleware/enrichment now.
+    // the objc version should accomodate them if it's really needed.
```

Comment:
```suggestion
    // TODO: make a note in the docs on this that we removed the old "options" property
    // and they need to write a middleware/enrichment now.
    // the objc version should accomodate them if it's really needed.
```

Is this comment still valid? If so, I suggest adding `TODO` to make it easier to find and fix in a future PR.

Reply:
This is the comment from Segment about this method, i copied it right as is so we can make a more informed decision.